### ISSUE TITLE
Avoiding upper case package names

### ISSUE THREAD

**Issue by anton-johansson** [speedment/speedment#6]
I personally don't like uppercase package names, and if I'm not entirely wrong, it's not recommended to have them either.

When I design my databases, I use a convention that matches the Java-classes, like so:

```
CREATE TABLE `Article` (
  `articleId` int(11) NOT NULL AUTO_INCREMENT,
  `articleNumber` varchar(16) NOT NULL,
  `articleName` varchar(100) NOT NULL,
  `description` varchar(255) NOT NULL,
  PRIMARY KEY (`articleId`),
  UNIQUE KEY `articleName_UNIQUE` (`articleName`),
  UNIQUE KEY `articleNumber_UNIQUE` (`articleNumber`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

It's the same for my database names. They can contain uppercase letters.

When generating my Speedment code, my package names will get uppercase letters. It would be nice if we could avoid that. :)

**Comment by Pyknic** [comment]
You are right, that looks like a bug. Generated code should follow the recommended formatting, which mean uppercase for classes and lowercase for packages, no matter how the database is defined.

**Comment by minborg** [comment]
I agree. However, this implies that you cannot have names that translates to the same lower case. I.e. schema "User" and "user" will collide.

**Comment by vladmihalcea** [comment]
On the Java side, it's better to use Java naming conventions and hide the database specific table/column naming in the generated code. That's what an ORM tool is supposed to to after all.

**Comment by Pyknic** [comment]
Database tables with uppercase names should not result in uppercase package names anymore. This is now fixed in the develop-branch and will be included in release 2.0.1.
